it/uk/gov/hmrc/vatregisteredcompanies/repository/PayloadBufferRepositoryISpec.scala

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatregisteredcompanies.repository
+
+import net.bytebuddy.pool.TypePool.Empty
+import uk.gov.hmrc.vatregisteredcompanies.helpers.IntegrationSpecBase
+import uk.gov.hmrc.vatregisteredcompanies.helpers.TestData._
+import uk.gov.hmrc.vatregisteredcompanies.repositories.PayloadWrapper
+
+class PayloadBufferRepositoryISpec extends IntegrationSpecBase {
+  override def beforeEach(): Unit = {
+    deleteAllBuffer
+  }
+
+  "Method: deleteAll" when {
+    "there are no records in the database" should {
+      "Have no records" in {
```

Comment:
I think instead of "Have no records" it should be do nothing, what do you think?